### DISCUSSION TITLE
roboshop / payment instance

### DISCUSSION THREAD

**Discussion by GCKeita** [learndevopsonline/learndevopsonline#1461] [Q&A] [answered]
I am wondering why all other parts of the app from cart to shipping... respond normal to 
sudo bash .sh but paytment is not responding.
All the configurations are same when launching instances
here is the erro code:

[ centos@payment ~/roboshop-shell ]$ sudo bash payment.sh
roboshop_app_password is missing

snapshot
![error code](https://github.com/learndevopsonline/learndevopsonline/assets/124757635/7a41240c-eeaf-4f5b-aaba-9a2805f34e03)

**Comment by sreedharm07** [comment]
Hi @GCKeita 
your server is down, can you please turn on your payment instance

**Reply by GCKeita** [reply]
it is up

**Reply by sreedharm07** [reply]
can you send your public ip  of payment server

**Reply by GCKeita** [reply]
52.87.182.22

**Reply by GCKeita** [reply]
52.87.182.22

**Reply by sreedharm07** [reply]
Hi @GCKeita  
you are running raghus repo thats why its throwing error. please clone your repo and run it will work.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/66792813-88bf-4fc2-8536-3e9ce1cca0eb)

**Reply by GCKeita** [reply]
I tried but unable to detache instance from raghu's repo
Any idea?

**Comment by GCKeita** [comment, marked as answer]
I was able to unclone and clone the right repo and now it works fine
Thank you [sreedharm07](https://github.com/sreedharm07)
